data/zilliqa/assets/zil1g42zl93x8cc9numpkxc0g0vyde485rt8k3shkd/meta.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "Gaymer",
   "symbol": "GAYMER",
+  "web": "https://gaymerchain.medium.com/what-is-gaymer-625a5b15f329"
```

Comment:
this is not a website, this should be in links.blog

Reply:
AxToken [https://axholding.medium.com/](url)

[https://viewblock.io/zilliqa/address/zil1rk9vdfu2xsp7y4h24qg78n6ss23mxxge5slsv2](url)

Reply:
excuse me sir, 
i'm sorry
i saw Ax Token build website like that, and it was approved